build/Dockerfile.bcc.builder

```diff
@@ -7,4 +7,4 @@ LABEL name=kepler-builder
 RUN yum install -y kernel-devel make git gcc rpm-build systemd && \
     yum clean all -y 
 
-RUN curl -LO https://go.dev/dl/go1.18.10.linux-amd64.tar.gz; mkdir -p /usr/local; tar -C /usr/local -xvzf go1.18.10.linux-amd64.tar.gz; rm -f go1.18.10.linux-amd64.tar.gz
+RUN curl -LO https://go.dev/dl/go1.21.1.linux-amd64.tar.gz; mkdir -p /usr/local; tar -C /usr/local -xvzf go1.21.1.linux-amd64.tar.gz; rm -f go1.21.1.linux-amd64.tar.gz
```

Comment:
can you make this a separate PR? The new builder image is needed for all other upgrade.